doc/install.rst

```diff
@@ -171,7 +171,7 @@ Installing PyGMT
 Now that you have GMT installed and your conda virtual environment activated,
 you can install PyGMT using any of the following methods:
```

Comment:
So far I have little to no knowledge about / experience with `mamba`. We are writing "conda virtual environment" at several occurrences, maybe we should remove "conda" to make this formulation more general?

Reply:
Agree that this might be confusing. However, although using `mamba`, the enviroments are technically still `conda` environments.  Maybe [this](https://mamba.readthedocs.io/en/latest/user_guide/mamba.html) makes it a little bit clearer.

Reply:
> maybe we should remove "conda" to make this formulation more general?

I think removing "conda" is a good choice.

Reply:
> > maybe we should remove "conda" to make this formulation more general?
> 
> I think removing "conda" is a good choice.

Done in cf506de0745f25718865c21a947e2a851f976801.

Reply:
At L126, can you also change `conda environment` to `virtual environment` please?:

```
Now we can create a new virtual environment with Python and all our dependencies
```